app/views/overview/_service-instance-row.html

```diff
@@ -152,6 +152,18 @@ <h3>
                 expandable="true"
                 linkify="true">
               </truncate-long-text></p>
+          </div>
+        </div>
+        <div ng-if="row.isMobileService && row.instanceStatus === 'ready'" class="row">
+          <div class="col-sm-6">
+            <div ng-repeat="mobileClient in row.filteredClients" class="col-sm-12 col-md-6 mobile-clients">
+              <span class="fa-2x {{mobileClient.metadata.annotations.icon}}"></span>
+              <a ng-href="{{mobileClient | navigateResourceURL}}">{{mobileClient.metadata.name}}</a>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-sm-12">
             <div ng-if="row.serviceClass.spec.externalMetadata.documentationUrl || row.serviceClass.spec.externalMetadata.supportUrl">
```

Comment:
can you please get it closer to the wirefarme https://www.dropbox.com/s/zjb4h64jo279654/Screenshot%202018-02-02%2011.29.07.png?dl=0
```
 icon   {{client.state.name}}
 icon   clientType/id or some other info
```

Reply:
This is the new look @witmicko 

![new-client-layout](https://user-images.githubusercontent.com/22113654/36670878-a16e3fec-1af1-11e8-8c80-e8577aafb6d1.png)

wyt?

Reply:
this is perfect @sedroche

Reply:
:+1:

Reply:
@sedroche is it possible to make the client names bold?